projects/agent-framework-javascript/agent-framework-javascript.md

```diff
@@ -0,0 +1,97 @@
+
+# Alignment with the OpenWallet Foundation Mission
+
+The objective is to promote interoperability by supporting numerous standards, as was outlined earlier. As a Hyperledger project, all components have been open source from the beginning.
+
+Maintainers will persist in collaborating with Standards Development Organizations (SDOs) across ecosystems, as done previously.
+
+# Code of Conduct
+
+The project currently follows the Hyperledger Code of Conduct as is the case for all Hyperledger projects: [Hyperledger Code of Conduct](https://wiki.hyperledger.org/display/HYP/Hyperledger+Code+of+Conduct). Hyperledger being part of the Linux Foundation too, it may be assumed its Code of Conduct is in line with the foundation's policies.
+
+# TAC Sponsor
+
+Tracy Kuhrt
+
+# Project License
+
+The project has an Apache 2.0 license: [Aries Framework JavaScript - License)](https://github.com/hyperledger/aries-framework-javascript%23license)
+
+# Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+- The framework [Aries Framework JavaScript)](https://github.com/hyperledger/aries-framework-javascript%23license)
+- The framework's extensions: [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext%23license)
+
+# Issue Tracker
+
+The issues are tracked using GitHub's Issues feature in the corresponding repository. For issues that span across multiple repositories, GitHub's Projects feature may be used.
+
+# External Dependencies
+
+Please see [Aries Framework JavaScript - Dependency Licenses](./dependency-licensing-overview.md)
+
+# Release Methodology
+
+All proposed repositories have continuous deployment/delivery pipelines built using [GitHub Actions](https://github.com/features/actions). The individual packages follow the [_semantic versioning_](https://semver.org/) method, ensuring consistency and safety. Whenever a pull request is merged into the `main` branch that doesn't cause a _major_, _minor_ or _patch_ version increase, an _alpha_ version is released.
+
+# Initial Maintainers
+
+| name               | Github                                                     |
+| ------------------ | ---------------------------------------------------------- |
+| Berend Sliedrecht  | [@blu3beri](https://github.com/blu3beri)                   |
+| Jakub Kočí         | [@jakubkoci](https://github.com/jakubkoci)                 |
+| James Ebert        | [@JamesKEbert](https://github.com/JamesKEbert)             |
+| Karim Stekelenburg | [@karimStekelenburg](https://github.com/karimStekelenburg) |
+| Timo Glastra       | [@TimoGlastra](https://github.com/TimoGlastra)             |
+| Ariel Gentile      | [@genaris](https://github.com/genaris)                     |
+| Jan Rietveld       | [@janrtvld](https://github.com/janrtvld)                   |
```

Comment:
Might be good to add affiliation to organizations here.

Reply:
My github handle has changed as well.